src/pyodide/internal/builtin_wrappers.js

```diff
@@ -147,7 +147,10 @@ export function patchFetch(origin) {
 
     // we didn't find it in the disk cache, continue with original fetch
     const response = await origFetch(url, options);
-    const arrayBuffer = await response.arrayBuffer();
+
+    const arrayBuffer = await new Response(response.body.pipeThrough(new DecompressionStream("gzip"))).arrayBuffer();
+
+    console.log("decompressed", fileName, arrayBuffer.byteLength, "bytes");
```

Comment:
Is the `console.log(...)` intentional here?

Reply:
Yeah, it's intentional, this whole section is not the final design and is just meant for testing for now.